src/main/resources/application.yml

```diff
@@ -38,6 +38,7 @@ kafdrop.monitor:
 
 kafka:
   brokerConnect: localhost:9092
+  isSecured: false
```

Comment:
Why is this added?

Reply:
I've checked it and removed as a redundant